src/main/resources/configCivcraft.yml

```diff
@@ -1543,6 +1545,39 @@ recipes:
       pumpkin pie:
         material: PUMPKIN_PIE
         amount: 192
+  Bind_Books:
+    type: PRODUCTION
+    name: Bind Books
+    production_time: 10s
+    input:
+      leather:
+        material: LEATHER
+        amount: 6
+      paper:
+        material: PAPER
+        amount: 18
+    output:
+      books:
+        material: BOOK
```

Comment:
This will produce empty, possibly unusable (due to missing nbt) written books, so kinda pointless